inst/tinytest/test_segfault.R

```diff
@@ -0,0 +1,9 @@
+
+library(Rblpapi)
+
+blpConnect()
+
+res <- Rblpapi:::bds_debug("YCSW0303 Index", "par_curve", repeats = 10000)
+
+print("no segfault")
+print(length(res))
```

Comment:
We need to either remove the test file (my suggestion) or condition it similar to the others.  Code can not simply access Blp, sadly.